### PR TITLE
Improving Adguard Home service

### PR DESCRIPTION
**Pull request by waschinski**
## Description

Changes to Adguard Home service:
- Showing `x.x% blocked` instead of the subtitle similar to how the PiHole service is doing it.
- The status text will no longer be `true/false` but `enabled/disabled/unknown`.
- Endpoints require to be logged in so fetch does now send cookies in order to prevent 403s.
- If data can't be fetched from endpoints a new status `unknown` will be shown.
 
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I've read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/main/CONTRIBUTING.md)
- [ ] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. 
- [ ] I have made corresponding changes the documentation (README.md).
- [x] I've checked my modifications for any breaking changes, especially in the `config.yml` file

### CODE REVIEW

**Comment by waschinski** [comment]
Any chance to get this merged anytime soon? It's been more than two months since I have created this PR and it is not really a huge change but it keeps me from using your repo.

File: src/components/services/AdGuardHome.vue [hunk 1]
```diff
@@ -16,14 +16,17 @@
...
+                </template>
+              </p>
             </div>
-            <div
-              v-if="status"
-              class="status"
-              v-bind:class="status.protection_enabled ? 'enabled' : 'disabled'"
-            >
-              {{ status.protection_enabled }}
+            <div v-if="!item.subtitle" class="status" :class="protection">
```
Comment (bastienwirtz) [inline src/components/services/AdGuardHome.vue:28]: Why the `v-if` here ? the status can be displayed with or without a subtitle.

**Review by bastienwirtz — COMMENTED** [review]
Hi @waschinski, 
sorry for the delay, and thank you for your work. I just tested it, It's perfect. 
I just got one question below and then we will merge that. 
Thanks for your patience.

File: src/components/services/AdGuardHome.vue (hunk 1 above)
Comment (waschinski) [reply src/components/services/AdGuardHome.vue]: You are absolutely right, the status is always being fetched anyway unlike the stats. I have just commited the code change accordingly.

**Comment by waschinski** [comment]
> Hi @waschinski,
> sorry for the delay, and thank you for your work. I just tested it, It's perfect.
> I just got one question below and then we will merge that.
> Thanks for your patience.

Thanks for getting back on this and sorry if I may have sounded harsh. By the way, kudos on the project, it's great!

**Comment by bastienwirtz** [comment]
No worries! thanks again :+1:

**Pull request merged**